test/models/workers_scaling/experiment_resources_interface_test.rb

```diff
@@ -228,10 +225,10 @@ def get_workers_stubs(amount)
         .with(anything, equals({cond: WorkersScaling::Query::Workers::INITIALIZING}))
         .returns(initializing_workers_records)
     @experiment.expects(:count_simulations_to_run).returns(simulations_to_run)
-    @resources_interface.stubs(:current_infrastructure_limit).returns(requested_amount)
+    @resources_interface.stubs(:current_resource_configuration_limit).returns(requested_amount)
     # when
     real_amount, _ = @resources_interface.send(:calculate_needed_workers, requested_amount,
```

Comment:
Ok, w takim razie nie rozumiem dlaczego testowane sa prywatne metody ? Albo metoda powinna byc publiczna albo jest to detal implementacyjny ktory nie powinien miec wplywu na system.